### PR TITLE
WIP: SSM parameter labels filter

### PR DESCRIPTION
**Pull request by pecigonzalo**
WORK IN PROGRESS

- Some entries are just "patched" to make `export` work:
`go run main.go export XYZService -l Current`
- I couldn't find what module/vendoring manager you are using or some instruction, so I just used temporarily `go mod`

Relates to #107

### CODE REVIEW

File: cmd/export.go [hunk 1]
```diff
@@ -33,6 +34,7 @@ var (
 
 func init() {
 	exportCmd.Flags().StringVarP(&exportFormat, "format", "f", "json", "Output format (json, java-properties, csv, tsv, dotenv)")
+	exportCmd.Flags().StringVarP(&labelFilter, "label", "l", "", "Output format (json, java-properties, csv, tsv, dotenv)")
```
Comment (jdub) [inline cmd/export.go]: Minor for a WIP, but important: Documentation string is copypasta.

**Comment by pecigonzalo** [comment]
Yes it is, and I known. I just copied a quick var to "show" it's possible
to do what was requested in the related issue.

Most of the code is just a hack.

Check that even the commit says WIP.

**Comment by nickatsegment** [comment]
@pecigonzalo Are you still working on this? Looks pretty out of date.

File: store/store.go [hunk 2]
```diff
@@ -62,7 +62,7 @@ type Store interface {
 	Write(id SecretId, value string) error
 	Read(id SecretId, version int) (Secret, error)
 	List(service string, includeValues bool) ([]Secret, error)
-	ListRaw(service string) ([]RawSecret, error)
+	ListRaw(service string, label string) ([]RawSecret, error)
```
Comment (nickatsegment) [inline store/store.go]: This breaks the API. Add a new func instead

**Comment by pecigonzalo** [comment]
@nickatsegment This was just a WIP as to show how to implement this, so I never finished it. Ill try and update it and complete it over the weekend.

**Comment by nickatsegment** [comment]
Ok, I'd like to close for now, just for tracking. When you've got something ready you want reviewed, just @ me and we can reopen.

**Comment by eyablonowitz** [comment]
@pecigonzalo wondering if you happened to take another look at this?  We might take a crack at it if you don't think you'll have time.

**Comment by pecigonzalo** [comment]
Go ahead, any work I have I have pushed in this PR as an example for
someone to be able to take over if I can't or serve as reference.

**Comment by DanielRedOak** [comment]
@eyablonowitz @pecigonzalo either of you made any further progress on this? I'm quite interested in this still but don't have the resourcing on my side to push it through ATM.

**Pull request closed without merging**